tests/test_policyai_rail.py

```
@@ -584,34 +592,24 @@ async def test_unsafe_with_missing_fields(monkeypatch):
+        assert result.is_blocked is True
+        assert result.metadata["assessment"] == "UNSAFE"
+        assert result.metadata["category"] == "Unknown"
+        assert result.metadata["severity"] == 0
+        assert result.reason == "Policy violation detected"
 
 
-def test_mapping_function_safe():
-    """Test the output mapping function returns False for SAFE."""
-    from nemoguardrails.library.policyai.actions import call_policyai_api_mapping
+def test_policyai_action_metadata_is_registration_only():
+    assert set(getattr(call_policyai_api, "action_meta")) == {
```

Ruff (0.15.21)

[warning] Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)